### DISCUSSION TITLE
Compare your stock portfolio to benchmarks - DOW, S&P500, ETFs etc.

### DISCUSSION THREAD

**Discussion by railsnoob** [afadil/wealthfolio#129] [Ideas]
Not many companies allow you to compare all your stocks with a benchmark nad help you figure out how to move from there to a diversified benchmark. 

Feature requrest:
1. Compare S&P to total port. 
2. Show how rebalancing on a single day every year would compare to S&P. 
3. Ideas for moving to be as diversified as benchmark. 
4.

**Comment by afadil** [comment]
This was introduced in version [1.0.22](https://github.com/afadil/wealthfolio/releases/tag/v1.0.22)

**Reply by railsnoob** [reply]
There seems to be a bug when I compare performances.
For some reason when I compare ADBE and VTI, there is a 14% difference at the beginning time for some reason. 

![Screenshot 2025-01-17 at 5 47 27 PM](https://github.com/user-attachments/assets/1da97bbe-277a-4f63-9307-ed2fe147af56)

**Reply by afadil** [reply]
Fixed in the latest versions
